src/lang/KclManager.ts

```diff
@@ -500,9 +502,13 @@ export class KclManager extends EventTarget {
 
   async safeParse(
     code: string,
-    wasmInstance?: ModuleType
+    providedWasmInstance?: ModuleType
```

Comment:
We should just get rid of this arg in the near future, why are callers responsible for maybe passing it in?